sdk/storage/ustorageclient/tests/azstorage_blob_ut/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(thisTestSuiteName azstorage_blob_ut)
+
+set(${thisTestSuiteName}_test_files
+    ${thisTestSuiteName}.c
+    ${PROJECT_SOURCE_DIR}/tests/${thisTestSuiteName}/main.c
+)
+
+set(${thisTestSuiteName}_c_files
+    ${PROJECT_SOURCE_DIR}/src/azstorage_blob.c
+)
+
+set(${thisTestSuiteName}_h_files
+    ${PROJECT_SOURCE_DIR}/inc/azstorage_blob_api.h
+)
+include_directories(${PROJECT_SOURCE_DIR}/inc)
+include_directories(${PROJECT_SOURCE_DIR}/config)
```

Comment:
this is a duplicate include directory after running azstorage_build_c_test_target right?